x-pack/plugin/build.gradle

```diff
@@ -104,39 +104,26 @@ integTestRunner {
   systemProperty 'tests.rest.blacklist', blacklist.join(',')
 }
 
-// location of generated keystores and certificates
+// location for keys and certificates
 File keystoreDir = new File(project.buildDir, 'keystore')
-
-// Generate the node's keystore
-File nodeKeystore = new File(keystoreDir, 'test-node.jks')
-task createNodeKeyStore(type: LoggedExec) {
-  doFirst {
-    if (nodeKeystore.parentFile.exists() == false) {
-      nodeKeystore.parentFile.mkdirs()
-    }
-    if (nodeKeystore.exists()) {
-      delete nodeKeystore
+File nodeKey = new File(keystoreDir, 'testnode.pem')
```

Comment:
This is usually written `file("$keystoreDir/testnode.pem")` in Gradle. It doesn't make a difference in this case, but sometimes does because `file()` resolves to `projectDir` but `new File()` resolves to working dir which can be the directory where the daemon was started. The `/` does work on Windows.

Reply:
Thanks, I was merely following existing conventions here. I'll change this